src/style.css

```diff
@@ -35,34 +35,43 @@ h4 {
 #about {
 	position: absolute;
 	bottom: 0;
-	right: 0;
-	margin: 0px 10px;
-	background-color: lightgrey;
-	border-radius: 4px 4px 0px 0px;
+	left: 0;
```

Comment:
I think your styling idea of making everything transparent is brilliant! One small problem, this about bar now overlaps with the case counter. I don't know exactly how you want to solve this design-wise but you should ;D